Add `LICENSE.md`

craciunoiuc opened the pull request:
This license is superseded by headers at the start of files. All files that miss a header have this one as the default.

Signed-off-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>

Closes #123

nderjung commented on License.md:
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2022, Unikraft OSS
```
```diff
- Copyright (c) 2022, Unikraft OSS
+ Copyright (c) 2022, The KraftKit Authors
```

nderjung reviewed (changes requested):
Thanks! Just two things, see inline.

Can you update the msg with:
```
doc: Add `LICENSE.md`

This license is superseded by headers at the start of files.
All files that miss a header have this one as the default.

GitHub-Fixes: #123
Signed-off-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>
```

craciunoiuc committed:
Add `LICENSE.md`

This license is superseded by headers at the start of files.
All files that miss a header have this one as the default.

GitHub-Fixes: #123
Signed-off-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>

nderjung reviewed (approved):
Awesome, thanks!
